Replace `find` command

DNEGEL3125 opened the pull request:
Use `fs.readdirSync` and `fs.rmSync` instead of `child_process.execFileSync` with `find` command for better maintainability and cross-platform compatibility.

With this change, `curlconverter.js` can be run on Mac.

0140454 commented on build/package-generators/curlconverter.js:
```diff
@@ -7,6 +7,7 @@ const path = require('path')
 const buildDir = execEnv.buildDir
 const packageJsonPath = path.join(buildDir, 'package.json')
 const packageLockPath = path.join(buildDir, 'package-lock.json')
+const generatorsDir = path.join(buildDir, "src/generators");
```
To make it more general, I would like to pass each path component as a separate argument.

```suggestion
const generatorsDir = path.join(buildDir, "src", "generators");
```

0140454 commented:
Oops, check failed :(

DNEGEL3125 committed:
Replace `find` command with fs operations for generator cleanup

Use `fs.readdirSync` and `fs.rmSync` instead of `child_process.execFileSync` with `find` command for better maintainability and cross-platform compatibility

DNEGEL3125 commented:
I updated `yarn.lock` by running `yarn install`. It should pass the build test now.
